src/meta_object.cpp

```diff
@@ -118,24 +96,24 @@ void AbstractMetaObjectBase::removeOwningClassLoader(const ClassLoader * loader)
 
 bool AbstractMetaObjectBase::isOwnedBy(const ClassLoader * loader) const
 {
-  const ClassLoaderVector & v = impl_->associated_class_loaders_;
+  const ClassLoaderVector & v = associated_class_loaders_;
   auto it = std::find(v.begin(), v.end(), loader);
   return it != v.end();
 }
 
 bool AbstractMetaObjectBase::isOwnedByAnybody() const
 {
-  return impl_->associated_class_loaders_.size() > 0;
+  return associated_class_loaders_.size() > 0;
 }
 
 size_t AbstractMetaObjectBase::getAssociatedClassLoadersCount() const
 {
-  return impl_->associated_class_loaders_.size();
+  return associated_class_loaders_.size();
 }
 
 ClassLoader * AbstractMetaObjectBase::getAssociatedClassLoader(size_t index) const
 {
-  return impl_->associated_class_loaders_[index];
+  return associated_class_loaders_[index];
```

Comment:
Prob should use `.at()`

Reply:
If you think so, would you like to take over this PR?